src/main/java/io/ark/core/responses/Fees.java

```diff
@@ -0,0 +1,42 @@
+package io.ark.core.responses;
+
+import static io.ark.core.util.Constants.ARKTOSHI;
+
+import org.json.JSONObject;
+
+import lombok.Data;
+
+/**
+ * This is a proposal for how to deal with all responses from the Ark Node.
+ * This is a frustrating case due to the unit conversions and flat JSON response. 
+ * Being the worse case scenario this is not too bad, and will allow us to use the futreGet added in HttpUtils.
```

Comment:
We could implement Jackson to take care of HTTP responses.

In a different project, I have all ark-node JSON extend from a base `NodeResponse` object. This makes casting much easier when you know what sort of response you will get back.

`NodeResponse votes = nodeManager.getVotes("AZse3vk8s3QEX1bqijFb21aSBeoF6vqLYE");`